plugins/available/java.plugin.bash

```diff
@@ -1,11 +1,11 @@
-cite about-plugin
+# shellcheck shell=bash
 about-plugin 'Java and JAR helper functions'
 
 function jar_manifest {
-  about "extracts the specified JAR file's MANIFEST file and prints it to stdout"
-  group 'java'
-  param '1: JAR file to extract the MANIFEST from'
-  example 'jar_manifest lib/foo.jar'
+	about "extracts the specified JAR file's MANIFEST file and prints it to stdout"
+	group 'java'
+	param '1: JAR file to extract the MANIFEST from'
+	example 'jar_manifest lib/foo.jar'
 
-  unzip -c $1 META-INF/MANIFEST.MF
+	unzip -c "${1?${FUNCNAME[0]}: manifest must be specified}" META-INF/MANIFEST.MF
```

Comment:
`$1:?` will trigger when $1 is provided but empty ie `""`

Reply:
Yes, that's correct. If there's no JAR/ZIP file to extract, it should show an error. If not, then `unzip` will show an error anyway.

...but, re-reading it I just realized it should say "JAR must be specified", not "manifest" 😆

Reply:
ok, fixed the wording.